projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -2951,6 +2971,9 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         super(_displayDensityOptions);
         this.locale = this.locale || this.localeId;
         this._transactions = this.transactionFactory.create(TRANSACTION_TYPE.None);
+        if (this.dataCloneStrategy) {
```

Comment:
no need for this check - `dataCloneStrategy` should be always set, otherwise there is an issue